src/main/resources/assets/memoryusagescreen/lang/ru_ru.json

```
@@ -1,8 +1,8 @@
 {
-  "text.autoconfig.memoryusagescreen.title": "Настройки Memory Usage Screen",
-  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на экране загрузки игры",
-  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира",
-  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Режим переключения отображения по нажатию",
+  "text.autoconfig.memoryusagescreen.title": "Конфигурация Memory Usage Screen",
+  "text.autoconfig.memoryusagescreen.option.enableInitLoadingScreen": "Включить отображение на начальном загрузочном экране.",
+  "text.autoconfig.memoryusagescreen.option.enableWorldLoadingScreen": "Включить отображение на экране загрузки мира.",
+  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Включить переключение режима отображения.",
```

Тут незнакомому с функционалом мода пользователю с первого прочтения вообще становится непонятно, о чём идёт речь. "Переключение режима отображения" – не интуитивная формулировка, не понятно, к чему она отсылает. Прежний перевод тоже был не самым понятным, поэтому предлагаю изменить на 
```suggestion
  "text.autoconfig.memoryusagescreen.option.enableToggleMode": "Активировать режим отображения по переключению",
```

---

Разработчики либо уточняют, что имеется ввиду, либо пишут коротко, тут написано коротко, либо пишут, как понимает он сам, забивая хрен на то, поймёт ли игрок или нет. Типа, страдайте, переводчики. Я написал так, как написал сам разработчик, а Активировать или включить — однотипные слова.